### PR TITLE
Tweak: Magma-ale and space beer are now a bottle subtype

### LINKED ISSUE #27354: Strange behavior with Magma Ale & Space Beer
### Exploit Reports

- [x] I confirm this issue is not an exploit. (Required)

### BYOND Version

515.1639

### Issue Description

So, apparantly, Space Beer and Magma Ale are both soda cans, and not glass bottles.
This has two knock-on effects.
- The first, and more obnoxious one, is that they cannot be smashed over someone else's head to make them flammable, in the same way as every other glass bottle.
- The second, and somewhat humorous one...is that you can crush them against your forehead when they're empty, producing a broken bottle object.

Image

### What did you expect to happen?

In the heat of the moment, I pull out the space beer I had prepared for the chaplain, and attempt to smash it over his head!

### What happened instead?

I give the chappy a sip of my beer.

### Why is this bad/What are the consequences?

It's inconsistent behavior, feels like an oversight in how cans/bottles are structured.

### Steps to reproduce the issue.

Find a booze-o-mat, grab a friend, have fun smashing everything in the list over their head. Find that you can't do that with the glass beer/ale bottles.

### When did the problem start happening?

I'd imagine since ever, this feels like a diamond problem/inheritance oversight.

### Extra information

So, cans apparently have an "is_glass" and "is_plastic" variable, but all it's used for is for determining what sound is played when the can is crushed/renaming the crushed can to be a broken bottle instead.

Image

I'm not too familiar with how this stuff is coded but this seems promising, though idk, for all I know this is refactor stuff.

### Relevant log output/runtime error

```DM

```

### PR DESCRIPTION
**Pull request by Toastical**
## What Does This PR Do
fixes #27354
Moves magma-ale and space beer to bottle from cans and updates the paths in code and maps. Upped the volume limit from 30 to 50.

## Why It's Good For The Game
Magma-ale and space beer now behave like glass bottles rather than soda cans. 

## Testing
Successfully forced a mob to drink on help intent
Smashed both bottles over mobs head in harm intent
Checked booze-o-mat bottles
Checked bar secure closet
Tested service borg emagged beer
Map tests:
- [x] Boxstation
- [x] Metastation
- [x] Deltastation
- [x] Cerestation
- [x] Emeraldstation
- [x] Lavaland outpost
- [x] Centcomm
- [x] Space  bar
- [x] Meatpackers
- [x] Moon outpost 19
- [x] Syndie Space Base
- [x] Syndie Depot
- [x] Althland Facility

- [x] Biodome Beach

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>

## Changelog

:cl:
tweak: Magma-ale and space beer are now bottles
tweak: Magma-ale and space beer bottles now hold 50u
/:cl:

### CODE REVIEW

**Comment by Toastical** [comment]
Uhh, I forgor that override_map will prevent it from checking other maps. I will fix them once im home.

**Review by silverplatedelta — COMMENTED** [review]
Code looks good, mapping pass is going to be required. 

I will note that this will remove the functionality of opening beer cans and hearing the "hiss" sound effect.

**Comment by Toastical** [comment]
> Code looks good, mapping pass is going to be required.
> 
> I will note that this will remove the functionality of opening beer cans and hearing the "hiss" sound effect.

Well, sprite-wise it wasn't a can to begin with. I could sprite beer cans and perhaps add an opening functionality to bottles? Like a cap pop for the beer bottle. A champagne pop for wine?

File: code/game/objects/structures/crates_lockers/closets/secure/bar.dm [hunk 1]
```diff
@@ -11,13 +11,13 @@
...
+	new /obj/item/reagent_containers/drinks/bottle/beer(src)
+	new /obj/item/reagent_containers/drinks/bottle/beer(src)
+	new /obj/item/reagent_containers/drinks/bottle/beer(src)
+	new /obj/item/reagent_containers/drinks/bottle/beer(src)
+	new /obj/item/reagent_containers/drinks/bottle/beer(src)
+	new /obj/item/reagent_containers/drinks/bottle/beer(src)
+	new /obj/item/reagent_containers/drinks/bottle/beer(src)
+	new /obj/item/reagent_containers/drinks/bottle/beer(src)
+	new /obj/item/reagent_containers/drinks/bottle/beer(src)
+	new /obj/item/reagent_containers/drinks/bottle/beer(src)
```
Comment (warriorstar-orion) [inline code/game/objects/structures/crates_lockers/closets/secure/bar.dm:23]:
```suggestion
	for(var/i in 1 to 10)
		new /obj/item/reagent_containers/drinks/bottle/beer(src)
```

**Review by warriorstar-orion — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/crates_lockers/closets/secure/bar.dm (hunk 1 above)
Comment (Toastical) [reply code/game/objects/structures/crates_lockers/closets/secure/bar.dm]: Oh, that's a whole lot sleeker. Damn

**Review by warriorstar-orion — APPROVED** [review]
Code LGTM

**Review by DGamerL — APPROVED** [review]

**Pull request merged**